### PR TITLE
VC-4242: added tables for banking module

### PR DESCRIPTION
**Pull request by at-kh**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: database/schema/13-banking.sql [hunk 1]
```diff
@@ -0,0 +1,74 @@
+-- +migrate Up
+CREATE TABLE IF NOT EXISTS vipcoin_chain_banking_banking (
```
Comment (IamStubborN) [inline database/schema/13-banking.sql]:
```suggestion
CREATE TABLE IF NOT EXISTS vipcoin_chain_banking_transfers (
```

File: database/schema/13-banking.sql [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    id                  SERIAL PRIMARY KEY NOT NULL,            -- banking id
+    asset               TEXT NOT NULL,                          -- banking asset
+    amount              BIGSERIAL,                              -- banking amount
+    kind                INT,                                    -- banking amount
```
Comment (IamStubborN) [inline database/schema/13-banking.sql]:
```suggestion
    kind                INT,                                    -- banking kind
```

File: database/schema/13-banking.sql [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    asset               TEXT NOT NULL,                      -- banking asset
+    amount              BIGSERIAL,                          -- banking amount
+    extras              JSONB                               -- banking extras
+);
+
+CREATE TABLE IF NOT EXISTS vipcoin_chain_banking_set_transfer_extra (
+    creator             TEXT NOT NULL,                      -- banking creator
+    id                  BIGSERIAL,                          -- banking id
+    extras              JSONB                               -- banking extras
+);
```
Comment (IamStubborN) [inline database/schema/13-banking.sql]: not need, we have 'extras' JSONB at every struct

File: database/schema/13-banking.sql [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    wallet_to           TEXT NOT NULL,                      -- banking wallet to
+    asset               TEXT NOT NULL,                      -- banking asset
+    amount              BIGSERIAL,                          -- banking amount
+    extras              JSONB                               -- banking extras
+);
+
+CREATE TABLE IF NOT EXISTS vipcoin_chain_banking_set_reward_manager_address (
+    creator            TEXT NOT NULL,                       -- banking creator
+    address            TEXT NOT NULL                        -- banking address
+);
```
Comment (IamStubborN) [inline database/schema/13-banking.sql:28]:
```sql
CREATE TABLE IF NOT EXISTS vipcoin_chain_banking_reward_manager_address (
    address            TEXT NOT NULL                        -- banking address
);
```

add table please, current state of `reward_manager_address` and `set_reward_manager_address` table - will be a history

**Review by IamStubborN — CHANGES_REQUESTED** [review]

**Comment by PavelVlasenkoGo** [comment]
LGTM

**Comment by IamStubborN** [comment]
LGTM

**Review by IamStubborN — APPROVED** [review]

**Pull request merged**